homeassistant/components/mobile_app/__init__.py

```diff
@@ -29,15 +29,19 @@
             DATA_BINARY_SENSOR: {},
             DATA_CONFIG_ENTRIES: {},
             DATA_DELETED_IDS: [],
-            DATA_DEVICES: {},
+            DATA_LOADED_ENTITIES: [],
+            DATA_LOADED_REGISTRATIONS: [],
+            DATA_REGISTRATIONS: {},
             DATA_SENSOR: {}
         }
 
     hass.data[DOMAIN] = {
         DATA_BINARY_SENSOR: app_config.get(DATA_BINARY_SENSOR, {}),
```

Comment:
Please for each key here, add documentation what data it holds, and what types key and value are.

Reply:
I also wonder if dict is still the best type to use here, or that you should just create a `MobileAppManager` class. Make one per app ID.